addons/mail/static/src/core/common/composer.js

```diff
@@ -212,6 +212,9 @@ export class Composer extends Component {
     }
 
     get placeholder() {
+        if (this.thread?.read_only) {
+            return _t("This channel is read only");
+        }
```

Comment:
Missing check for admin? -> centralized logic might help

Reply:
Admin or not, you still need some indication that the channel is read-only, and this placeholder is the only one that will do that for an admin.